mgmtsystem_action/__openerp__.py

```diff
@@ -20,31 +20,25 @@
 ##############################################################################
 {
     "name": "Management System - Action",
-    "version": "8.0.1.2.0",
+    "version": "9.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Management System",
-    "description": """\
-This module enables you to manage the different actions of your management
-system:
-  * immediate actions
-  * corrective actions
-  * preventive actions
-  * improvement opportunities.
-""",
-    "depends": ['mgmtsystem', 'crm_claim'],
+    "depends": ['mgmtsystem', 'crm_claim', 'mail'],
```

Comment:
I have removed dependance on crm.claim, and added dependance on mail.thread so that we will still have chatter integration. i then added The needed field in mgmtsystem_action model. is ot what you expected